Makefile

```
@@ -0,0 +1,84 @@
+#                                                                              #
+#                +--------------------------------------------+                #
+#                |   █████╗ ██╗   ██╗██████╗  █████╗ ███████╗ |                #
+#                |  ██╔══██╗██║   ██║██╔══██╗██╔══██╗██╔════╝ |                #
+#                |  ███████║██║   ██║██████╔╝███████║█████╗   |                #
+#                |  ██╔══██║██║   ██║██╔══██╗██╔══██║██╔══╝   |                #
+#                |  ██║  ██║╚██████╔╝██║  ██║██║  ██║███████╗ |                #
+#                |  ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝ |                #
+#                +--------------------------------------------+                #
+#                                                                              #
+#                         Distributed Systems Runtime                          #
+#                                                                              #
+# ---------------------------------------------------------------------------- #
+#                                                                              #
+#   Licensed under the Apache License, Version 2.0 (the "License");            #
+#   you may not use this file except in compliance with the License.           #
+#   You may obtain a copy of the License at                                    #
+#                                                                              #
+#       http://www.apache.org/licenses/LICENSE-2.0                             #
+#                                                                              #
+#   Unless required by applicable law or agreed to in writing, software        #
+#   distributed under the License is distributed on an "AS IS" BASIS,          #
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   #
+#   See the License for the specific language governing permissions and        #
+#   limitations under the License.                                             #
+#                                                                              #
+# ---------------------------------------------------------------------------- #
+
+all: compile
+
+# Variables and Settings
+version     ?=  0.0.1
+target      ?=  ae
+org         ?=  aurae-runtime
+authorname  ?=  The Aurae Authors
+authoremail ?=  info@aurae.io
+license     ?=  Apache2
+year        ?=  2023
+copyright   ?=  Copyright (c) $(year)
+
+compile: mod ## Compile for the local architecture ⚙
+	@echo "Compiling..."
+	go build -ldflags "\
+	-X 'github.com/$(org)/$(target).Version=$(version)' \
+	-X 'github.com/$(org)/$(target).AuthorName=$(authorname)' \
+	-X 'github.com/$(org)/$(target).AuthorEmail=$(authoremail)' \
+	-X 'github.com/$(org)/$(target).Copyright=$(copyright)' \
+	-X 'github.com/$(org)/$(target).License=$(license)' \
+	-X 'github.com/$(org)/$(target).Name=$(target)'" \
```

```suggestion
	-X 'main.Version=$(version)' \
	-X 'main.AuthorName=$(authorname)' \
	-X 'main.AuthorEmail=$(authoremail)' \
	-X 'main.Copyright=$(copyright)' \
	-X 'main.License=$(license)' \
	-X 'main.Name=$(target)'" \
```
Since these are now in `package main`, they would be set as above.